process/block/postprocess/oneMBPostProcessor.go

```diff
@@ -113,9 +118,14 @@ func (opp *oneMBPostProcessor) CreateAllInterMiniBlocks() []*block.MiniBlock {
 // VerifyInterMiniBlocks verifies if the receipts/bad transactions added to the block are valid
 func (opp *oneMBPostProcessor) VerifyInterMiniBlocks(body *block.Body) error {
 	scrMbs := opp.CreateAllInterMiniBlocks()
-	createdMapMbs := make(map[uint32]*block.MiniBlock)
+	createdMapMbs := make(map[uint32][]*block.MiniBlock)
 	for _, mb := range scrMbs {
-		createdMapMbs[mb.ReceiverShardID] = mb
+		_, ok := createdMapMbs[mb.ReceiverShardID]
```

Comment:
this is duplicated code with the one in intermediateResults.go - you could move this map creation to a base func

Reply:
done